tests/org.codechecker.eclipse.rcp.unit.tests/resources/CodeChecker/bin/CodeChecker

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python
+import sys
```

Comment:
Please add a docstring to the script.
It is used to emulate some CodeChecker commands for the tests.